Source/SwiftLintFramework/Rules/Idiomatic/TrailingSemicolonRule.swift

```diff
@@ -65,8 +66,8 @@ private extension TrailingSemicolonRule {
             }
 
             correctionPositions.append(node.positionAfterSkippingLeadingTrivia)
-            // Is there a better way to remove a node? Should we somehow keep trailing trivia?
-            return super.visit(TokenSyntax(.semicolon, presence: .missing))
+            // Keep the trailing trivia
+            return ("" as TokenSyntax).with(\.trailingTrivia, node.trailingTrivia ?? .zero)
```

Comment:
Not sure If this is the best way to do this

Reply:
Not sure either. This seems to boil down to `.identifier("")` which is a bit strange. Do you like `.unknown("").with(\.trailingTrivia, ...)` more?

Reply:
I think so - it feels less hacky than the cast, so I went with that.